Modules/Microsoft.PowerShell.UnixCompleters/Microsoft.PowerShell.UnixCompleters/ZshUtilCompleter.cs

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Management.Automation;
+using System.Management.Automation.Language;
+using System.Reflection;
+using System.Text;
+
+namespace Microsoft.PowerShell.UnixCompleters
+{
+    public class ZshUtilCompleter : IUnixUtilCompleter
+    {
+        private static readonly string s_completionScriptPath = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "zcomplete.sh");
```

Comment:
Is `Path.GetDirectoryName` really needed?

Reply:
Is is not? If we have the absolute path of a file and try to `Combine` it with the name of another file I think they both end up in the path:

```powershell
> [System.IO.Path]::Combine("$PWD\here.txt", "there.txt")
C:\Users\Robert Holt\Documents\Dev\Microsoft\vscode-PowerShell\here.txt\there.txt
```

Reply:
For Assembly.Location docs say "Gets the full path or UNC location of the loaded file". So I'd expect:
```suggestion
        private static readonly string s_completionScriptPath = Path.Combine(Assembly.GetExecutingAssembly().Location, "zcomplete.sh");
```

Reply:
> Gets the full path or UNC location of the loaded file

Yes, this is a sibling of that assembly file. So we must remove the end